Cargo.toml

```diff
@@ -106,13 +106,15 @@ prettytable-rs = "0.10"
 rayon = "1"
 regex = "1"
 reqwest = { version = "0.11", default-features = false }
+rstest = "0.19.0"
 semver = "1"
 serial_test = "2"
 serde = "1.0"
 serde_json = "1.0"
 serde_json_traversal = "0.2"
 serde_yaml = "0.9"
 shell-candy = "0.4"
+speculoos = "0.11.0"
```

Comment:
do these need to be in the normal deps or just dev deps?

Reply:
All of the dependency versions are declared here, whether they're dev-dependencies or not. We declare whether they are normal dependencies or dev-dependencies below.